packages/mobile/src/components/lineup/types.ts

```diff
@@ -67,7 +67,7 @@ export type LineupProps = {
    * A header to display at the top of the lineup,
    * will scroll with the rest of the content
    */
-  header?: SectionListProps<any>['ListHeaderComponent']
+  header?: SectionListProps<unknown>['ListHeaderComponent']
```

Comment:
why does this have to be unknown? (certainly better than any, but just curious)

Reply:
yeah just updated to unknown to be safer, but ultimately that type-arg should be the value of our `section` const, which we can def determine and add here if we feel the need